packages/rum/src/boot/startRecording.spec.ts

```diff
@@ -165,6 +165,37 @@ describe('startRecording', () => {
     })
   })
 
+  it('full snapshot related records should have the view change date', (done) => {
+    const clock = mockClock()
+    const { lifeCycle } = setupBuilder.build()
+
+    changeView(lifeCycle)
+    flushSegment(lifeCycle)
+
+    waitRequestSendCalls(2, (calls) => {
+      readRequestSegment(calls.first(), (segment) => {
+        expect(segment.records).toEqual([
+          { type: RecordType.Meta, timestamp: Date.now(), data: jasmine.any(Object) },
+          { type: RecordType.Focus, timestamp: Date.now(), data: jasmine.any(Object) },
+          { type: RecordType.FullSnapshot, timestamp: Date.now(), data: jasmine.any(Object) },
+          { type: RecordType.VisualViewport, timestamp: Date.now(), data: jasmine.any(Object) },
+          { type: RecordType.ViewEnd, timestamp: Date.now() },
+        ])
+        clock.cleanup()
+
+        readRequestSegment(calls.mostRecent(), (segment) => {
+          expect(segment.records).toEqual([
+            { type: RecordType.Meta, timestamp: 1, data: jasmine.any(Object) },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Use a constant like `VIEW_TIMESTAMP = 1` to be more explicit

Reply:
sure